cw/capture.py

```diff
@@ -1410,6 +1430,11 @@ def capture_otbn_vertical_batch(ot, ktp, capture_cfg, scope_type):
         project.traces.tm.setTraceSegmentStatus(s, True)
     assert len(project.traces) == capture_cfg["num_traces"]
 
+    # Save metadata to project file
+    pll_frequency = round(scope._scope.clock.clkgen_freq // scope._scope.clock.adc_mul, -6)
+    sample_rate = int(round(scope._scope.clock.clkgen_freq, -6))
+    save_metadata(project, capture_cfg, cycles, pll_frequency, sample_rate)
```

Comment:
Could you add `save_metadata` to other capture functions? Currently, if I try to run tvla on e.g. kmac data I get an error because the project doesn't have metadata. 

Note: I think that `cycles` argument will have to be `None` in all function calls other than otbn-vertical

Reply:
Sure! I'll add it to all the other capture functions.
The error you get is probably caused by accessing metadata for the plot title (as you already pointed out below). So I assume that after solving this issue, KMAC TVLA should work again.
But nevertheless, it's a good idea to include this in all the other capture functions.